packages/form-js-viewer/src/render/components/form-fields/Checklist.js

```diff
@@ -0,0 +1,92 @@
+import { useContext } from 'preact/hooks';
+
+import { FormContext } from '../../context';
+
+import Description from '../Description';
+import Errors from '../Errors';
+import Label from '../Label';
+
+import {
+  formFieldClasses,
+  prefixId
+} from '../Util';
+
+const type = 'checklist';
+
+
+export default function Checklist(props) {
+  const {
+    disabled,
+    errors = [],
+    field,
+    value = [],
+  } = props;
+
+  const {
+    description,
+    id,
+    label,
+    values
+  } = field;
+
+  const toggleCheckbox = (v) => {
+
+    var newValue = [...value];
+
+    if (!newValue.includes(v)) {
+      newValue.push(v);
+    } else {
+      newValue = newValue.filter(x => x != v);
+    }
+
+    props.onChange({
+      field,
+      value: newValue,
+    });
+  };
+
+  const { formId } = useContext(FormContext);
+
+  return <div class={ formFieldClasses(type, errors) }>
+    <Label
+      label={ label } />
+    {
+      values.map((v, index) => {
```

Comment:
Personally, I wouldn't use such small variable names if the variable is heavily used. But this probably comes from that `value` is already given via `prop`.

Reply:
Yeah exactly, and it is something that we could have a look at again with the whole 'options' discussion we started.